src/frontend/src/expr/mod.rs

```diff
@@ -46,6 +46,7 @@ pub use order_by_expr::{OrderBy, OrderByExpr};
 mod expr_mutator;
 mod expr_rewriter;
 mod expr_visitor;
+mod function;
```

Comment:
nit: Maybe name it `function_impl` or other clearer name. Having `function_call` (for the expr variant definition) and `function` as siblings inside `frontend::expr` can be confusing.